DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Utility.cs

```diff
@@ -0,0 +1,174 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.DeID.SharedLib.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    /// <summary>
+    /// Utility functions are used to parse DICOM data including DA, DT, AS. The format for these VR is given in DICOM standard.
+    /// http://dicom.nema.org/medical/Dicom/2017e/output/chtml/part05/sect_6.2.html
+    /// </summary>
+    public static class Utility
```

Comment:
>Utility [](http://example.com/codeflow?start=24&length=7)

should we move this class into some kinds of shared lib? @moria97 @qfyin @sowu880 ? Just a suggestion. :blush:

Reply:
It is DICOM related utility, not belongs to shared lib. 
Changing these functions to extension may be a good way.